soc/arm/st_stm32/stm32f0/dts_fixup.h

```diff
@@ -119,8 +119,7 @@
 #define DT_CAN_1_CLOCK_BUS		DT_ST_STM32_CAN_40006400_CLOCK_BUS
 #define DT_CAN_1_CLOCK_BITS		DT_ST_STM32_CAN_40006400_CLOCK_BITS
 
-#define DT_FLASH_DEV_BASE_ADDRESS	DT_ST_STM32F0_FLASH_CONTROLLER_40022000_BASE_ADDRESS
-#define DT_FLASH_DEV_NAME		DT_ST_STM32F0_FLASH_CONTROLLER_40022000_LABEL
+#define DT_FLASH_DEV_NAME		DT_INST_0_ST_STM32F0_FLASH_CONTROLLER_LABEL
```

Comment:
The `flash_stm32.c` driver is using `DT_INST_0_SOC_NV_FLASH_LABEL` as a flash name
```
DT_INST_0_SOC_NV_FLASH_LABEL="FLASH_STM32"
```
however the `DT_FLASH_DEV_NAME` is mapped to the flash controller name
```
DT_INST_0_ST_STM32F0_FLASH_CONTROLLER_LABEL="FLASH_CTRL"

```
The two don't match. This is causing run time error in MCUBoot.

Reply:
@mnkp, thanks for reporting. I'm fixing it.